scripts/check-changed.mjs

```
@@ -0,0 +1,128 @@
+}
+
+function changedSourceTsFiles(baseRef) {
+  return changedFilesSince(baseRef).filter(
+    (fileName) => fileName.startsWith('src/') && fileName.endsWith('.ts')
+  );
+}
+
+const changedTsFiles = changedSourceTsFiles(options.baseRef);
+
+if (!changedTsFiles.length) {
+  console.log(`No changed src/**/*.ts files found since ${options.baseRef}.`);
```

ESLint

[error] 'console' is not defined.

(no-undef)